src/Columns/ColumnVector.h

```diff
@@ -240,6 +240,14 @@ class ColumnVector final : public COWHelper<ColumnVectorHelper, ColumnVector<T>>
         data.push_back(static_cast<T>(x.get<T>()));
     }
 
+    bool tryInsert(const DB::Field & x) override
+    {
+        NearestFieldType<T> value;
+        if (!x.tryGet<NearestFieldType<T>>(value))
```

Comment:
So, `tryInsert` doesn't insert `Int32` into `ColumnVector<Int64>`, is it desired behavior?